apps/system/utils/README.md

```diff
@@ -140,27 +144,19 @@ Options:
  -i PRINT_INTERVAL     Show cpuload values every PRINT_INTERVAL(s)
  -n ITERATION_COUNT    Iterate showing cpuload values ITERATION_COUNT times
 
-TASH>> cpuload
-Started CPU monitor with interval 5.
-PID | Pri |    2s |
+TASH>>cpuload
+
+--------------------------------------------------
+Started CPU monitor with interval 5 sec.
+TASH>>Non-Zero CPU utilization (2s)
```

Comment:
Can you pls check what is 2s and what is 5 sec here? 
As i understand. we will display the cpu load info every 5 secs. 
At the end of every 2 secs the accumulated ticks value is divided by 2. This is an internal aspect of the cpuload utility to prevent overflow of data. The user does not need to know this information. So, we can remove it.

Reply:
i have updated the line with "Non-Zero CPU utilization trend (updated every 5 sec)" because we will see the estimated average value after every 5 sec (or cpuload interval for showing data)